grails-app/controllers/au/org/ala/volunteer/TutorialsController.groovy

```diff
@@ -352,21 +358,33 @@ class TutorialsController {
         }
 
         def institutionFilter = []
-        Institution institution = (params.institutionFilter ? Institution.get(params.long('institutionFilter')) : null)
+        def institutionId = params.long('institutionFilter') ?: getSessionFilter("institutionFilter")
+        Institution institution = institutionId ? Institution.get(institutionId as long) : null
+
         if (institution) {
             institutionFilter.add(institution)
-            updateSessionFilter([institutionFilter: params.institutionFilter])
+            params.institutionFilter = institutionId
+            updateSessionFilter([institutionFilter: institutionId])
         } else {
             institutionFilter = institutionList
             removeSessionFilter("institutionFilter")
         }
 
-        def statusFilter = (params.statusFilter ?: null)
-        if (params.statusFilter) updateSessionFilter([statusFilter: params.statusFilter])
-        else removeSessionFilter("statusFilter")
+        def statusFilter = params.statusFilter ?: getSessionFilter("statusFilter")
+        if (statusFilter) {
+            params.statusFilter = statusFilter
+            updateSessionFilter([statusFilter: statusFilter])
+        } else {
+            removeSessionFilter("statusFilter")
```

Comment:
Is this going to ever be called?  Maybe if there is a falsey value that's not null in the session filter? e.g. ""?